src/components/LoanCards/AlgoliaLoanCardAdapter.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<div class="algolia-loan-card-adapterm column column-block">
+		<GridLoanCard
+			:key="loan.id"
+			:loan="adaptedLoan"
+			:is-visitor="true"
+			:items-in-basket="itemsInBasket" />
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import GridLoanCard from '@/components/LoanCards/GridLoanCard';
+
+export default {
+	components: {
+		GridLoanCard
+	},
+	props: {
+		loan: {
+			type: Object,
+			default: () => {}
+		},
+		itemsInBasket: {
+			type: Array,
+			default: () => []
+		},
+	},
+	computed: {
+		adaptedLoan() {
+			// TODO: There must be a better way
+			const defaultImage = `https://www-dev-kiva-org.global.ssl.fastly.net/img/w480h360/${_get(this.loan, 'image.hash')}.jpg`; // eslint-disable-line
+			const retinaImage = `https://www-dev-kiva-org.global.ssl.fastly.net/img/w960h720/${_get(this.loan, 'image.hash')}.jpg`; // eslint-disable-line
+			const exprirationDate = new Date(_get(this.loan, 'plannedExpirationDate') * 1000);
```

Comment:
We have the [date-fns](https://date-fns.org/) library included. It might have something to make this easier.

Reply:
totally would. I think the next step will be to refactor this adapter to serve either a grid loan card or the new list loan card so there is def more work.